src/components/controls/ProbeKeySelector.svelte

```diff
@@ -8,24 +8,24 @@
   export let options;
   export let currentKey;
   export let active;
-  
+
```

Comment:
Not a big deal, but for the time being, it may make sense to turn off the editor settings that are removing trailing spaces and adding newlines at the ends of files, just to make the diffs a little bit less noisy.